lib/resty/dns/resolver.lua

```diff
@@ -81,6 +83,7 @@ local resolver_errstrs = {
     "refused",          -- 5
 }
 
+local soa_names = { "serial", "refresh", "retry", "expire", "mininum"}
```

Comment:
Also, I think `soa_rr_fields` is a better variable name than `soa_names`. The latter is confusing.